src/mlpack/methods/reinforcement_learning/environment/pendulum.hpp

```diff
@@ -93,13 +93,24 @@ class Pendulum
    * @param maxAngularVelocity Maximum angular velocity.
    * @param maxTorque Maximum torque.
    * @param dt The differential value.
+   * @param angleThreshold The region about the upright position where the
+   *    state is considered terminal.
+   * @param maxTimeSteps The number of time steps after which the episode
+   *    terminates. If the value is 0, there is no limit.
    */
   Pendulum(const double maxAngularVelocity = 8,
            const double maxTorque = 2.0,
-           const double dt = 0.05) :
+           const double dt = 0.05,
+           const double angleThreshold = M_PI / 12,
+           const double doneReward = 0.0,
+           const size_t maxTimeSteps = 0) :
```

Comment:
Not sure about the name, do you think ``maxSteps`` or ``maxIterations`` could work as well?